WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +725,62 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
```

Comment:
I don't think the name of this method matches what this method exactly does. Technically, we are loading the post from remote, but we don't actually want it to be called to fetch a post, but instead used to update a conflicted post with the remote version of it. I'd suggest renaming it to something like `updateConflictedPostWithItsRemoteVersion` or something similar which conveys the intention of the implementation better.

Reply:
nice change there, more descriptive of what/why is going on, addressed in 4c92554e11